src/uct/cuda/cuda_copy/cuda_copy_iface.c

```diff
@@ -392,11 +392,12 @@ static ucs_mpool_ops_t uct_cuda_copy_event_desc_mpool_ops = {
 };
 
 static uct_iface_internal_ops_t uct_cuda_copy_iface_internal_ops = {
-    .iface_estimate_perf = uct_cuda_copy_estimate_perf,
-    .iface_vfs_refresh   = (uct_iface_vfs_refresh_func_t)ucs_empty_function,
-    .ep_query            = (uct_ep_query_func_t)ucs_empty_function_return_unsupported,
-    .ep_invalidate       = (uct_ep_invalidate_func_t)ucs_empty_function_return_unsupported,
-    .ep_connect_to_ep_v2 = ucs_empty_function_return_unsupported
+    .iface_estimate_perf   = uct_cuda_copy_estimate_perf,
+    .iface_vfs_refresh     = (uct_iface_vfs_refresh_func_t)ucs_empty_function,
+    .ep_query              = (uct_ep_query_func_t)ucs_empty_function_return_unsupported,
+    .ep_invalidate         = (uct_ep_invalidate_func_t)ucs_empty_function_return_unsupported,
+    .ep_connect_to_ep_v2   = ucs_empty_function_return_unsupported,
+    .iface_is_reachable_v2 = (uct_iface_is_reachable_v2_func_t)ucs_empty_function_return_unsupported
```

Comment:
doesn't it break anything that this function is unsupported?

Reply:
Since wireup.c still calls v1's interface it wouldn't break something. Once we support v2 function in all components, v2 will be used to replace v1 in wireup.c. Thanks.